src/libsyntax/config.rs

```diff
@@ -365,7 +353,3 @@ impl<'a> MutVisitor for StripUnconfigured<'a> {
         noop_visit_pat(pat, self)
     }
 }
-
-fn is_cfg(attr: &ast::Attribute) -> bool {
```

Comment:
Why this change?

Reply:
Drive-by cleanup.

Reply:
Yeah but I mean... it doesn't seem like cleanup but rather the opposite? Why not offer this function as an abstraction?

Reply:
Ok, will revert.